db/schema/version.go

```diff
@@ -24,7 +24,8 @@ func (Version) Fields() []ent.Field {
 	return []ent.Field{
 		field.String("mod_id"),
 		field.String("version").MaxLen(16),
-		field.String("sml_version").MaxLen(16),
+		field.String("sml_version").MaxLen(16).Optional(),
```

Comment:
Does this field even need to exist anymore? Can't we add a `version_dependency` to the SML version for all existing versions now?